src/wwwroot/startupscripts/startagent-win.cmd

```diff
@@ -49,3 +49,7 @@ if not "%LASTEXITCODE%" == "0" (
     echo "Agent disconnected successfully, exiting"
     exit /b 0
 )
+
+rmdir /S /Q %WORKSPACEPATH%
+echo "Requesting reboot to kill all processes"
+%HELIX_PYTHONPATH% -c "from helix.workitemutil import request_reboot; request_reboot('Optional reason string')"
```

Comment:
Please fill out 'optional reason string' or delete it.

Reply:
Also, if you're not going to make requesting reboot conditional, can probably get rid of the workspace path cleanup since the reboot will kill running stuff and the next work item will remove it.

Reply:
Removed rmdir and added reason string